client/src/components/Card/Card.css

```diff
@@ -46,18 +46,16 @@
 }
 .Card--extra-small {
   color: var(--white);
-  min-width: 180px;
+  min-width: 7rem;
+  max-width: 7rem;
+  min-height: 5rem;
```

Comment:
hey, do we need to set explicit min and max widths and heights? How does that hold up with responsive?

Reply:
@Mdusseau did you miss this comment?